ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/DockerStatic/Dockerfiles/Dockerfile.sles12

```diff
@@ -0,0 +1,38 @@
+FROM registry.suse.com/suse/sles12sp5:latest
+
+RUN zypper ar https://download.opensuse.org/distribution/leap/15.4/repo/oss/ sles15oss
+RUN zypper --gpg-auto-import-keys refresh
+RUN zypper update -y && zypper install -y wget perl openssh-server unzip zip tar gzip
+
+RUN wget 'https://api.adoptium.net/v3/binary/latest/11/ga/linux/s390x/jdk/hotspot/normal/eclipse?project=jdk' -O /tmp/jdk11.tar.gz
```

Comment:
any reason we use 11/ga/linux/s390x ?

Reply:
Our other dockerstatic dockerfiles use jdk 8, but we dont release that for s390x, so I've used 11